.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+Make sure these boxes are checked!
```

Comment:
let's add something like "thanks so much!" and like an emoji or two. i would also add the lines:

```
$ rustup override set nightly
$ rustup component add rustfmt-preview --toolchain nightly
```

Reply:
Sounds good. Throwing in emojis and instructions is easy to do!